pkgs/development/octave-modules/general/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildOctavePackage
+, lib
+, fetchurl
+, pkg-config
+, nettle
+}:
+
+buildOctavePackage rec {
+  pname = "general";
+  version = "2.1.1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/octave/${pname}-${version}.tar.gz";
+    sha256 = "0jmvczssqz1aa665v9h8k9cchb7mg3n9af6b5kh9b2qcjl4r9l7v";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    nettle
```

Comment:
In that case, what should be in `nativeBuildInputs` rather than `buildInputs`? I'm not quite sure where certain tools are supposed to go. I know that the main difference comes down to cross-compilation, but am unsure beyond that. (This goes for all the other comments (below) regarding this particular issue.